src/m365/util/commands/accesstoken/accesstoken-get.ts

```diff
@@ -60,10 +60,14 @@ class UtilAccessTokenGetCommand extends Command {
       }
     }
 
-    try { 
+    if (resource.toLowerCase() === 'graph') {
+      resource = auth.defaultResource;
+    }
+
+    try {
```

Comment:
Let's move this `try` up so the entire `commandAction` function is covered by it.

Reply:
Done.